dev/io.openliberty.java.internal_fat/publish/servers/java17-cdi-server/bootstrap.properties

```diff
@@ -0,0 +1,12 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
Please update the copyright date since this is a new file.

Reply:
Done